### PR TITLE
Fix: Permissions styling

### LINKED ISSUE #2440: Modify permissions tab - text not aligned or centred
Per the following screenshot, the text in the table is not horizontally aligned together and vertically it should be centered 
permissions tab alignment

### PR DESCRIPTION
**Pull request by Onokaev**
## Overview
Fixes #2440 

### Demo

[screen-recorder-wed-mar-08-2023-12-27-47.webm](https://user-images.githubusercontent.com/45680252/223727344-e1325244-96c5-4769-adf9-1c62dcdb4d97.webm)

[screen-recorder-wed-mar-08-2023-16-32-28.webm](https://user-images.githubusercontent.com/45680252/223727359-3a7750de-33f1-497c-acbd-bb1db60acd87.webm)

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Click on the Modify Permissions tab
Adjust your browser size and confirm that all items are aligned as per requirement
Sign in and out while interacting witht the tab

### CODE REVIEW

File: src/app/views/query-runner/request/permissions/Permission.tsx [hunk 1]
```diff
@@ -138,43 +136,44 @@ export const Permission = (permissionProps?: IPermissionProps): JSX.Element => {
...
-        <Label><FormattedMessage id='Yes' /></Label>
+      return <div style={{paddingRight: '28px', textAlign: 'center' }}>
+        <span>{translateMessage('Yes')}</span>
       </div>;
     } else {
-      return <div style={{ textAlign: 'left', paddingLeft: '10px' }}>
-        <Label><FormattedMessage id='No' /></Label>
+      return <div style={{paddingRight: '28px', textAlign: 'center' }}>
+        <span>{translateMessage('No')}</span>
       </div>;
```
Comment (thewahome) [inline src/app/views/query-runner/request/permissions/Permission.tsx:145]: Can we have the styles defined? they are identical, they can be defined in a style definition above where they are used

File: src/app/views/query-runner/request/permissions/Permission.tsx [hunk 2]
```diff
@@ -138,43 +136,44 @@ export const Permission = (permissionProps?: IPermissionProps): JSX.Element => {
...
     }
   }
 
   const consentedButton = (consented: boolean, item: any, hostId: string): JSX.Element => {
     if (consented) {
       if(userHasRequiredPermissions()){
-        return <PrimaryButton onClick={() => handleRevoke(item)} style={{width: '100px', textAlign:'left'}}>
+        return <PrimaryButton onClick={() => handleRevoke(item)}
+          styles={{ root: {width: '100%' } }}
```
Comment (thewahome) [inline src/app/views/query-runner/request/permissions/Permission.tsx:153]: similarly this and line 168 are also identical

File: src/app/views/query-runner/request/permissions/Permission.tsx [hunk 3]
```diff
@@ -196,15 +195,15 @@ export const Permission = (permissionProps?: IPermissionProps): JSX.Element => {
   const permissionConsentTypeLabel = (permissionInAllPrincipal : boolean) : JSX.Element => {
     if(permissionInAllPrincipal){
       return (
-        <div style={{textAlign: 'left', paddingLeft: '10px'}}>
-          <Label>{translateMessage('AllPrincipal')}</Label>
+        <div style={{textAlign: 'center', paddingLeft: '10px'}} aria-label='AllPrincipal'>
```
Comment (thewahome) [inline src/app/views/query-runner/request/permissions/Permission.tsx:198]: The aria-label could also benefit from translation :)

And the styling is repeated.

File: src/app/views/query-runner/request/permissions/Permission.tsx [hunk 4]
```diff
@@ -272,13 +271,14 @@ export const Permission = (permissionProps?: IPermissionProps): JSX.Element => {
...
         fieldName: 'isAdmin',
-        minWidth:  150,
-        maxWidth:  200,
+        minWidth:  180,
         ariaLabel: translateMessage('Administrator permission'),
         isMultiline: true,
         headerClassName: 'permissionHeader',
         styles: columnCellStyles,
-        onRenderHeader: () => renderColumnHeader('Admin consent required')
+        onRenderHeader: () => renderColumnHeader('Admin consent required'),
```
Comment (thewahome) [inline src/app/views/query-runner/request/permissions/Permission.tsx:279]: Should this be translated?

Comment (Onokaev) [reply src/app/views/query-runner/request/permissions/Permission.tsx:279]: Yes. They are translated in the renderColumnHeader implementation

File: src/app/views/query-runner/request/permissions/Permission.tsx (hunk 1 above)
Comment (Onokaev) [reply src/app/views/query-runner/request/permissions/Permission.tsx:145]: I have lifted all the styles to the .styles file. Nice observation

**Review by thewahome — DISMISSED** [review]

File: src/app/views/query-runner/request/permissions/Permission.tsx [hunk 5]
```diff
@@ -142,43 +140,44 @@ export const Permission = (permissionProps?: IPermissionProps): JSX.Element => {
...
+        <span>{translateMessage('Yes')}</span>
       </div>;
     } else {
-      return <div style={{ textAlign: 'left', paddingLeft: '10px' }}>
-        <Label><FormattedMessage id='No' /></Label>
+      return <div style={adminLabelStyles}>
+        <span>{translateMessage('No')}</span>
       </div>;
     }
   }
```
Comment (thewahome) [inline src/app/views/query-runner/request/permissions/Permission.tsx:151]: Last comment:

```
const adminLabel = (item: any): JSX.Element => {
    return <div style={{ textAlign: 'left', paddingLeft: '10px' }}>
      <Label>{item.isAdmin ? <FormattedMessage id='Yes' /> : <FormattedMessage id='No' />}</Label>
    </div>;
  }
  ```

**Pull request merged**